management-api-server/src/main/resources/openapi-configuration.json

```diff
@@ -2,89 +2,20 @@
     "resourcePackages": [
         "com.datastax.mgmtapi.resources"
     ],
-    "prettyPrint": true,
-    "cacheTTL": 0,
     "openAPI": {
         "info": {
-            "version": "0.1",
+            "version": "0.1.66",
```

Comment:
This will be where the verssion change to 0.1.67 has to happen (removing the other comment)